backend/btrixcloud/orgs.py

```diff
@@ -253,6 +253,15 @@ async def get_max_pages_per_crawl(self, org: Organization):
         """Return org-specific max pages per crawl setting or 0."""
         return await get_max_pages_per_crawl(self.orgs, org.id)
 
+    async def inc_bytes_stored(self, org: Organization, size: int):
```

Comment:
Is there a reason to have this call to the non-class function?

Previously, added non-class functions so can use in operator, which didn't have the ops classes, but now that they do, should probably just keep things in the class?

Reply:
oh, i see, i guess the OrgOps isn't yet available everywhere that its called, can do a cleanup pass on this another time, so feel free to disregard for now.